schemas/UserProperties.json

```diff
@@ -59,6 +59,21 @@
           {"const": "Group", "description": "Modern layout."},
           {"const": "Compact", "description": "Modern layout with compact option enabled."}
         ]
+      },
+      "verificationState": {
+        "type": "string",
+        "oneOf": [
+          {"const": "Verified", "description": "The device is considered to be verified, it has been signed by its user identity."},
+          {"const": "NotVerified", "description": "The device is unverified."}
+        ]
+      },
+      "recoveryState": {
+        "type": "string",
```

Comment:
Can you add a description please? Can be copy paste of what's in CryptoSessionState.json.